devops/perf-tests-job-template.yml

```diff
@@ -25,8 +25,19 @@ jobs:
   - script: pip install -r requirements.txt
     displayName: 'Install required packages'
 
+  - script: pip install azureml-sdk
+    displayName: 'Install azureml-sdk'
+
   - script: python -m pytest -s test/perf --junitxml=./TEST--TEST.xml -o junit_suite_name="$(Agent.JobName)"
     displayName: 'Run Perf tests'
+    env:
+      SERVICE_PRINCIPAL_PASSWORD: $(fairlearnautomationspsecret)
+      SERVICE_PRINCIPAL_ID: $(fairlearnautomationspid)
+      TENANT_ID: $(fairlearnautomationsptenantid)
+      SUBSCRIPTION_ID: cecafb73-04ae-4432-9f96-d96925d28058
```

Comment:
Can the subscription/rg/ws etc. be made into template parameters?

Reply:
They sure can!